crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -223,7 +223,13 @@ pub(crate) async fn dialog_export_doc_w_prefs(appwindow: &RnAppWindow, canvas: &
                                 log::error!("exporting document failed, Error: `{e:?}`");
                                 appwindow.overlays().dispatch_toast_error(&gettext("Exporting document failed"));
                             } else {
-                                appwindow.overlays().dispatch_toast_text(&gettext("Exported document successfully"), crate::overlays::TEXT_TOAST_TIMEOUT_DEFAULT);
+                                appwindow.overlays().dispatch_toast_w_button_singleton(&gettext("Exported document successfully"), &gettext("View in file manager"), clone!(@weak canvas, @weak appwindow => move |_reload_toast| {
+                                    let folder_path = file.parent().expect("the file should always have a parent directory").path().unwrap();
+                                    if let Err(e) = open::that(folder_path.to_str().unwrap()) {
+                                        appwindow.overlays().dispatch_toast_error(&gettext("Failed to open the file in the file manager"));
+                                        log::debug!("opening the parent folder in the file manager failed: {e:?}",);
```

Comment:
Change this to `log::error` and include the file name in the error message

Reply:
Fixed by newest commit